### PR TITLE
Support OR in query tool

### PR DESCRIPTION
**Pull request by loicgasser**
[Test Link](https://mf-geoadmin3.dev.bgdi.ch/gal_and_or_query/?lang=fr&topic=aviation)

This is to address:
https://github.com/geoadmin/mf-geoadmin3/issues/2356

TODO: One translation

Relies on https://github.com/geoadmin/mf-chsdi3/pull/1688

### CODE REVIEW

**Comment by gjn** [comment]
This looks fine to me. Need translations for `query_operator` in translation sheet.

**Comment by cedricmoullet** [comment]
@oterral could you please review ?

File: src/components/query/QueryDirective.js [hunk 1]
```diff
@@ -74,6 +75,16 @@ goog.require('ga_storage_service');
       $scope.updateOp(idx, filter);
     };
 
+    $scope.queryOperators = [{
+      label: 'AND',
+      value: 'and'
+    }, {
+      label: 'OR',
+      value: 'or'
```
Comment (oterral) [inline src/components/query/QueryDirective.js]: Shouldn't we translate the labels?

**Comment by oterral** [comment]
Missing translations otherwise lgtm.

File: src/components/query/QueryDirective.js (hunk 1 above)
Comment (loicgasser) [inline src/components/query/QueryDirective.js]: I think it's fine, what would you put instead of OR and AND?

Comment (oterral) [inline src/components/query/QueryDirective.js]: "Et " et "Ou"

**Pull request merged**